### PR TITLE
Add nixduino

### PR DESCRIPTION
**Pull request by boredom101**
Added nixduino, a tool to build arduino sketches using nix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +92,10 @@
 
 ## Programming Languages
 
+### Arduino
+
+* [nixduino](https://github.com/boredom101/nixduino) - Nix based tool to help build arduino sketches.
```
Comment (houstdav000) [inline README.md:97]: Two small changes to the description:

- Add a hyphen/dash between "Nix" and "based".
- Arduino [should be capitalized](https://www.arduino.cc/en/Main/Policy) when used (had to look that up).

**Review by houstdav000 — CHANGES_REQUESTED** [review]
The entry itself looks pretty good, just a few small changes to match other entries and "official" naming schemes.

As for the project, the `README.md` could use some fleshing out, particularly on "how to use". The examples you provide are both pretty simple, so you could probably include one of them and annotations on it in the `README.md`, and could keep another, more complex example in the examples folder.

**Comment by boredom101** [comment]
I fixed the pull request and added an annotated example to the nixduino readme

**Review by houstdav000 — APPROVED** [review]
The fixes look good, and the project repo I feel is a lot better with that in-`README.md` example.

I'd say it's good to merge.

**Pull request merged**